### DISCUSSION TITLE
Which city was 🇯🇵’s capital for more than 1000 years?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#870] [Galaxy Trivia] [answered]
**Bonus:** What is the capital now and in what year did it become?

**Comment by peterbaccaro** [comment, marked as answer]
The city that served as Japan's capital for more than 1000 years is **Kyoto**. From 794 to 1868, Kyoto was the imperial capital of Japan. During this period, it was the centre of political, cultural, and religious activities in the country. 

**Tokyo** is the capital now and became the capital of Japan in **1868**.  In 1868, the capital was officially moved to Tokyo (formerly known as Edo) as part of the Meiji Restoration.
